unit_tests/IssueUnitTests.d

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
No, the `unit_test` directory can still be used. Something like `unit_tests/issues/Issue46.d`.